### ISSUE TITLE
Tooltips Provided for the "Chart" are not descriptive enough: A11y_.NET 9 template_.NET 9 template_Chart_Non text content.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26800]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG1.1.1; #SH_.NET9template_MAC_dec2024; #DesktopApp; #.NET Core; #NonTextContent; #A11yMAS; #MAC;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version: 15.2

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Observe that Tooltips Provided for the "Chart" are not descriptive enough.

### Actual Result:
Tooltips Provided for the "Chart" are not descriptive enough

### Expected Result:
Tooltips Provided for the "Chart" should be more descriptive so user will understand it's purpose.

### User Impact:
It is important to ensure that the tooltips are descriptive. If they are not, all users may experience difficulties and be adversely impacted. Providing clear and detailed tooltips will enhance user experience and prevent potential issues.

### Attachment:

https://github.com/user-attachments/assets/6499af31-0e46-4c72-aee5-23ca5739ae49

**Comment by InduPriya1805** [comment]
Rev:Inse

**Comment by sheiksyedm** [comment]
Duplicate #26260

**Comment by AnKushSingh05** [comment]
> Duplicate [#26260](https://github.com/dotnet/maui/issues/26260)





Hi @sheiksyedm, the bug you provided is for Win32, while the logged bug is for the MAC platform, so they are not duplicates.

**Comment by sheiksyedm** [comment]
These changes are done in the below PR:
https://github.com/dotnet/maui-samples/pull/569
